kolibri/plugins/html5_viewer/kolibri_plugin.py

```diff
@@ -7,13 +7,18 @@
 from kolibri.core.content import hooks as content_hooks
 from kolibri.plugins import KolibriPluginBase
 from kolibri.plugins.hooks import register_hook
+from kolibri.utils.conf import OPTIONS
 
 
 class HTML5AppPlugin(KolibriPluginBase):
-    pass
+    kolibri_options = "options"
 
 
 @register_hook
 class HTML5AppAsset(content_hooks.ContentRendererHook):
     bundle_id = "main"
     presets = (format_presets.HTML5_ZIP,)
+
+    @property
+    def plugin_data(self):
+        return {"sandbox": OPTIONS["HTML5"]["SANDBOX"]}
```

Comment:
Should we perhaps also add this to the H5P renderer?

Reply:
And if so, what's the cleanest way to share this stuff across plugins without copypasta?

Reply:
I think no, for now - and we also need to work out how we share other code between the two, so I'd rather work that out before doing any duplication.